lib/src/components/Banner.tsx

```diff
@@ -9,13 +9,71 @@ type TBanner = {
 	image?: string,
 	primaryButton?: ReactElement,
 	secondaryButton?: ReactElement,
-	children: ReactElement | ReactElement[],
+	children?: ReactElement | ReactElement[],
 	canClose?: boolean,
 	onClose?: () => void,
 	onClick?: () => void,
-	maxHeight?: string,
 	variant?: 'default' | 'image' | 'split' | 'background'
 }
+type TDefaultVariant = {
+	title?: string,
+	primaryButton?: ReactElement,
+	secondaryButton?: ReactElement,
+	children?: ReactElement | ReactElement[],
+	variant?: 'default' | 'image' | 'split' | 'background'
+}
+
+function	SplitVariant({image}: {image: string}) {
+	return (
+		<div className={'w-full md:w-1/2 relative'}>
+			<div
+				className={'flex md:hidden overflow-hidden relative -mx-1 -mt-1 w-full h-full rounded-xl border-x-2 border-b-2 border-primary max-h-48'}
```

Comment:
`flex md:hidden` => Display by default as flex, but if screen is md or bigger, move to display hidden. AKA only display that on small devices, aka mobile